setup.cfg

```diff
@@ -36,6 +36,14 @@ install_requires =
   scipy >= 0.19
   setuptools >= 41.2
 
+[options.packages.find]
+exclude =
+  plasmapy.examples
+  plasmapy.examples.*
```

Comment:
So we don't want `examples` to be importable, and this ensures that? Cool.

Reply:
This will exclude `examples` from the distribution all together.  I discovered this when doing the native namespace packages.  Since we plan to move the `examples` to the root directory, I figured I leave this here.

However, I discovered a little while back that if we exclude examples, then it will be a little more difficult for users to get their hands on the example data (e.g. the traces for langmuir probes). I prefer the idea of not distributing example data with the main `plasmapy` distribution, but we should consider some mechanism that makes it easy for a user to get his or her hands on the data. Maybe something like `plasmapy.utils.get_example_data()`. ¯\\_(ツ)_/¯  Or, we could place the data in a separate distribution that could be an extra dependency during install, like `pip install plasmapy[examples]`.